public class ExceptionHandlingExample {
    public static void main(String[] args) {
        int[] numbers = { 1, 2, 3 };

        try {
      
            System.out.println("Trying to access element at index 3: " + numbers[3]);
        } catch (ArrayIndexOutOfBoundsException e) {
           
            System.out.println("Exception caught: ArrayIndexOutOfBoundsException");
        } finally {
         
            System.out.println("Finally block executed.");
        }

     
        System.out.println("Program continues...");
    }
}
